src/languages/tidb/tidb.functions.ts

```diff
@@ -0,0 +1,295 @@
+export const functions: string[] = [
```

Comment:
For some reason you have excluded aggregate functions like `count()`, `avg()`, etc. I'm pretty confident that TiDB supports these (like all other SQL dialects I'm aware of).

You've also excluded `cast()`.

All of these are probably treated internally special syntax and therefore not listed under functions. But for the purposes of the formatter, we want to format them all similarly.

If you had run the test-suite you would have seen lots of tests failing because of these missing functions.

I'll add these.